css/api.scss

```diff
@@ -41,7 +41,14 @@
     background-color: $smartData-color;
   }
 }
-
+#description {
+  .markdown-body {
+    border: 0 solid black;
```

Comment:
Why define styles for a border when you set its width to `0`? Why not simply use `none`?